src/components/common/TextField.tsx

```
@@ -0,0 +1,59 @@
+    return (
+      <div className="flex flex-col mx-auto w-83.75">
+        <div
+          className={cn(
+            "relative flex w-full items-center border-b pb-2 transition-colors",
+            WRAPPER_VARIANT_STYLES[variant]
+          )}
+        >
+          <input
+            ref={ref}
+            className={cn(
+              "w-full bg-transparent outline-none body-2 caret-white",
```

ESLint

[error] Replace `w-full·bg-transparent·outline-none·body-2·caret-white` with `body-2·w-full·bg-transparent·caret-white·outline-none`

(prettier/prettier)